### DISCUSSION TITLE
Using HTTP requests as a Destination

### DISCUSSION THREAD

**Discussion by AndryHTC** [ConduitIO/conduit#1112] [Connectors]
Hello Conduit community,

I am Andrea, a Conduit user based in Italy. I'm currently exploring different features of Conduit and I have run into a question that I hope you could help me with.

I've been trying to find an option within Conduit that would allow me to use HTTP requests as a destination. I've searched through the documentation and forums but haven't been able to find a clear answer to this. This is a bit surprising for me as I had expected this to be a common feature in a system like Conduit.

So, my question is: Is it possible to use HTTP requests as a destination within Conduit? If so, could you guide me on how to accomplish this?

If this is not currently a feature, is there any plan to include it in future updates or is there a workaround I could use?

I would appreciate any information or guidance on this matter.

Thank you all in advance for your time and support.

Andrea

**Comment by hariso** [comment]
@AndryHTC Thank you for your interest in Conduit! We're glad to have users in Italy as well.:)

So you'd basically like to be able to send data to an HTTP endpoint? The closest we have is an HTTP processor, which allows a record's payload to be replaced with the response from an HTTP endpoint, but depending on your case that's probably not it.

We do have it in our backlog, #506. I'll bring this up to the team and then we'll let you know what are the plans there.

Thanks again for reaching out!

**Reply by AndryHTC** [reply]
@hariso Thank YOU!

**Reply by hariso** [reply]
@AndryHTC Letting you know that our HTTP connector is done! You can find it [here](https://github.com/conduitio-labs/conduit-connector-http). It features a source, a destination, and the possibility to "program" the source (you can write JavaScript code to customize the request and parse the response).

**Reply by AndryHTC** [reply]
That's really great.

I'll keep it in mind the next time I'll need data integration
